ui_testing/testcases/basic_tests/test05_testplans.py

```diff
@@ -446,41 +470,57 @@ def test013_create_testplans_same_name_different_materialtype(self):
 
         data = self.test_plan.search(testplan_name)
         self.assertGreaterEqual(len(data), 2)
-        
-    def test014_create_testplans_same_name_materialtype_all_article(self):
-        '''
-        LIMS-3500
-        New: Test plan: Creation Approach: I can't create two test plans
-        with the same name & same materiel type & one with any article
-        and the other one all
-        '''
-
-        testplans = self.test_plan_api.get_all_test_plans_json()
-        testplan = random.choice(testplans)
-        
-        testplan_name = self.test_plan.create_new_test_plan(material_type=testplan['materialType'],
-                                                            article=(testplan['article'])[0])
-        self.base_selenium.LOGGER.info(
-            'New testplan is created successfully with name: {}, article name: {} and material type: {}'.format(
-                testplan_name, (testplan['article'])[0], testplan['materialType']))
-
-
-        self.base_selenium.LOGGER.info(
-            'Attempting to create another testplan with the same name & material type as the previously created one,'
-            ' and all articles')
 
-        # create another testplan with the same data
-        self.test_plan.create_new_test_plan(name=testplan_name, material_type=testplan['materialType'],
-                                            article='All')
+    def test015_update_quantification_limit_new_version_created(self):
+        """
+        New: Test plan: Limits of quantification Approach: In case I update the limits
+        of quantification this will trigger new version in the active table & version table
+        LIMS-4426
+        """
+        # select random test plan and git material type
+        testplan = random.choice(self.test_plan_api.get_inprogress_testplans())
+        testplan_name = testplan['testPlanName']
+        material_type = testplan['materialType']
+        old_version = testplan['version']
+        # create quantitative test unit with quantification limit
+        self.test_unit_page.get_test_units_page()
+        new_name = self.generate_random_string()
+        new_method = self.generate_random_string()
+        new_random_limit = self.generate_random_number()
+        self.test_unit_page.create_quantitative_testunit(name=new_name, material_type=material_type, category='',
+                                                         upper_limit=new_random_limit, lower_limit=new_random_limit,
+                                                         spec_or_quan='quan', method=new_method)
+
+        self.test_unit_page.save(save_btn='general:save_form', logger_msg='Save new testunit')
+        # navigate to the chosen test plan edit page
+        self.test_plan.get_test_plans_page()
+        self.test_plan.get_test_plan_edit_page(testplan_name)
+        # navigate to the test units selection tab
+        self.test_plan.set_test_unit(test_unit=new_name)
+        self.test_plan.sleep_tiny()
```

Comment:
Use API to add a test unit to an existing test plan

Reply:
I couldn't use it